pages/users/[id].tsx

```diff
@@ -182,44 +182,48 @@ export default function User({ loginContext }: Props) {
       <main className="bg-ifpink flex-1 justify-center flex pt-16 pb-32">
         <div style={{ flexBasis: 1138 }}>
           <OffsetBorderContainer>
-            <div className="px-24 pt-16 pb-12">
+            <div className="px-7 pt-7 md:px-24 md:pt-16 pb-12">
```

Comment:
I hope you don't mind that I left 2 different sets of paddings (fixed the pt for desktop though), it's more aligned with the designs (and saves space on mobile)